### ISSUE TITLE
[flow-resolvers] Flow Plugin not defining some types in generated file

### ISSUE THREAD

**Issue by javedbartlett** [dotansimha/graphql-code-generator#5322]
**Describe the bug**
 
For some reason 3 graphql types are being generated without actually being defined. Here they are:
![image](https://user-images.githubusercontent.com/48427864/103182723-2e4a6b00-4873-11eb-8e54-58dc4a0640d0.png)
![image](https://user-images.githubusercontent.com/48427864/103182726-373b3c80-4873-11eb-8c0d-4a0367ff6818.png)

For some reason they are not being defined. Are these possibly TS types?

**To Reproduce**
Use the same yml file as me, create a basic backend schema like I did and when you generate you'll see that some types are created that are never defined.
<!--
#############
#############
🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 

PLEASE READ THE FOLLOWING: 
You can use this template to create a live sandbox of the issue: 
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template
Make sure sure fork this template, and run `yarn generate` in the terminal. 
Please make sure the Codegen and plugins version under `package.json` matches yours. 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ ISSUES WITH A LIVE REPRODUCTION WILL BE ANSWERED QUICKLY ~~~~~~~
~~~~~~ ISSUES WITHOUT REPRODUCTION MIGHT GET CLOSED WITHOUT RESPONSE ~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#############
#############
-->

1. My GraphQL schema:

```graphql
export const bookDef = gql`
  type Book {
    title: String
    author: String
  }

  type Query {
    books: [Book]
  }
`;
```

2. My GraphQL operations:

```graphql
query Books {
  books {
    title
    author
  }
}
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: "http://localhost:4000/graphql"
documents: "src/graphql/**/*.graphql"
generates:
  src/generated/graphql.js:
    plugins:
      - "flow"
      - "flow-resolvers"
      - "flow-operations"
```

**Expected behavior**
It should define Maybe, GraphQLScalarType and GraphQLScalarTypeConfig 

**Environment:**

- OS: Mac OS 11.1
- `@graphql-codegen/...`: 
-    @graphql-codegen/cli": "1.20.0",
    "@graphql-codegen/flow": "^1.18.2",
    "@graphql-codegen/flow-operations": "^1.18.5",
    "@graphql-codegen/flow-resolvers": "^1.17.13",
    "@graphql-codegen/introspection": "1.18.1",
- NodeJS: 15.3.0

**Additional context**
<!-- Add any other context about the problem here. -->

**Comment by dotansimha** [comment]
All issues here seems to come from `flow-resolvers`, right? or is it from the other plugins as well?

**Comment by pinglinh** [comment]
I am having this exact issue as well and I am using these packages:

- "@graphql-codegen/cli": "2.6.1"
- "@graphql-codegen/flow": "2.2.4"
- "@graphql-codegen/flow-resolvers": "2.2.4"

Yeah I think this is coming from flow-resolvers

**Comment by charlypoly** [comment]
Hi!

@Tohaker provided a fix that has been shipped in `@graphql-codegen/flow-resolvers@2.2.6`.

Thank you for your contribution @Tohaker! 🙇🏼
